articles/vpn-gateway/vpn-gateway-vpn-faq.md

```diff
@@ -67,6 +67,15 @@ Policy-based gateways implement policy-based VPNs. Policy-based VPNs encrypt and
 
 Route-based gateways implement the route-based VPNs. Route-based VPNs use "routes" in the IP forwarding or routing table to direct packets into their corresponding tunnel interfaces. The tunnel interfaces then encrypt or decrypt the packets in and out of the tunnels. The policy or traffic selector for route-based VPNs are configured as any-to-any (or wild cards).
 
+### Can I update my Policy-based VPN gateway to Route-based?
+No. An Azure Vnet gateway type cannot be changed from policy-based to route-based or the other way. The gateway must be deleted and recreated, a process taking around 60 minutes. The IP address of the gateway will not be preserved nor will the Pre-Shared Key (PSK).
+1. Delete any connections associated with the gateway to be deleted.
+2. Delete the gateway:
+* [Using the portal](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-delete-vnet-gateway-portal)
+* [Using PowerShell](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-delete-vnet-gateway-powershell)
+* [Using classic Powershell](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-delete-vnet-gateway-classic-powershell)
+3. [Create a new gateway of desired type and complete the VPN setup](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-site-to-site-resource-manager-portal#VNetGateway)
+
```

Comment:
Hi! I can approve this if you change the links to this format:

[Azure portal](vpn-gateway-delete-vnet-gateway-portal.md)
[Azure PowerShell](vpn-gateway-delete-vnet-gateway-powershell.md)
[Azure PowerShell](classic)](vpn-gateway-delete-vnet-gateway-classic-powershell.md)
[Create a new gateway and complete the VPN setup](vpn-gateway-howto-site-to-site-resource-manager-portal.md#VNetGateway)